src/Cake.Common/Tools/DotNetCore/DotNetCoreSettings.cs

```diff
@@ -15,5 +15,15 @@ public abstract class DotNetCoreSettings : ToolSettings
         /// Gets or sets a value indicating whether to not enable verbose output.
         /// </summary>
         public bool Verbose { get; set; }
```

Comment:
verbose is removed in the [executor PR](https://github.com/cake-build/cake/pull/1551/files#diff-3d8f3306c17edd46c61b555365e3ab2bL62) as it was the one using it. I felt that removing verbose here would add a lot of non common changes to a pr that is meant to only introduce changes that are common to all PRs. If you still disagree let me know and I'll remove the changes